src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/CSharpInstructionDecoder.cs

```diff
@@ -34,6 +34,18 @@ private CSharpInstructionDecoder()
             AddMemberOptions(SymbolDisplayMemberOptions.IncludeParameters).
             WithParameterOptions(SymbolDisplayParameterOptions.IncludeType);
 
+        internal static readonly SymbolDisplayFormat s_indexerCompactNameFormat = CompactNameFormat.
+            WithMemberOptions(SymbolDisplayMemberOptions.IncludeParameters);
```

Comment:
nit: may be worth a comment or making `SymbolDisplayParameterOptions.None` explicit for clarity